src/App.js

```diff
@@ -1,16 +1,40 @@
-import React from 'react';
+import React,{useState} from 'react';
 import './App.css';
-import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import chatMessages from './data/messages.json'
 
 const App = () => {
+  const [entries, setEntries] = useState(chatMessages)
```

Comment:
👍 We need a piece of state to track changes to our messages, initialized from the json file we loaded.